src/geckomat/gather_kcats/runDLKcatFromDocker.m

```diff
@@ -0,0 +1,111 @@
+%   DLKcatFile      path to the DLKcat.tsv file (including file name), as
+%                   written by writeDLKcatFile. Once DLKcat is succesfully
+%                   run, the DLKcatFile will be overwritten with the DLKcat
+%                   output. Optional, otherwise the file location will be
+%                   assumed to be in the model-specific 'data' sub-folder
+%                   taken from modelAdapter (e.g.
+%                   GECKO/userData/ecYeastGEM/data/DLKcat.tsv)
+%   modelAdapter    a loaded model adapter. (Optional, will otherwise use
+%                   the default model adapter)
+%   DLKcatPath      path where DLKcat is/will be installed. (Optional,
+%                   defaults to GECKO/dlkcat)
+%
+%   NOTE: 1. Requires Docker to be installed. Visit "https://www.docker.com"
+%         2. Runtime will depend on whether the image is to be downloaded or not.
+
+%Get the GECKO path
+geckoPath = findGECKOroot();
+
+if nargin < 4 || isempty(DLKcatPath)
+    DLKcatPath = fullfile(geckoPath,'dlkcat');
+end
+
+if nargin < 3 || isempty(modelAdapter)
+    modelAdapter = ModelAdapterManager.getDefaultAdapter();
+    if isempty(modelAdapter)
+        error('Either send in a modelAdapter or set the default model adapter in the ModelAdapterManager.')
+    end
+end
+params = modelAdapter.params;
+
+if nargin < 2 || isempty(DLKcatFile)
+    DLKcatFile = fullfile(params.path,'data','DLKcat.tsv');
+end
+
+if nargin < 1 || isempty(DLKcatPath)
+    deleteImage = false;
+end
+
+%% Check and install requirements
+% On Mac, docker might not be properly loaded if MATLAB is started via
+% launcher and not terminal.
+if ismac
+    setenv('PATH', strcat('/usr/local/bin', ':', getenv("PATH")));
+end
+
+% Check if docker is installed
+[checks.docker.status, checks.docker.out] = system('docker --version');
+if checks.docker.status ~= 0
+    error('Cannot find Docker. Make sure it is installed')
+end
+
+% Dowload DLKcat package
+if ~exist(fullfile(DLKcatPath,'DLKcat.py'),'file')
+    if ~exist(fullfile(DLKcatPath),'dir')
+        mkdir(fullfile(DLKcatPath));
+    end
+    disp('=== Downloading DLKcat ...')
+    packageURL = 'https://github.com/SysBioChalmers/GECKO/raw/dlkcatPackage/dlkcat.zip';
+    %packageURL = 'https://github.com/SysBioChalmers/GECKO/releases/download/v3.0.0/dlkcat_package.zip';
+    websave(fullfile(DLKcatPath,'dlkcat_package.zip'),packageURL);
+    unzip(fullfile(DLKcatPath,'dlkcat_package.zip'),geckoPath);
+    delete(fullfile(DLKcatPath,'dlkcat_package.zip'));
+end
+
+currPath = pwd();
+cd(DLKcatPath);
+
+% First check if the Docker image exist
+[checks.image.status, checks.image.out] = system('docker images');
+if checks.image.status == 0 && ~contains(checks.image.out,'dlkcat')
+    % Check if the image file is already dowloaded
+    if ~exist(fullfile(DLKcatPath,'dlkcat_docker.tar.gz'),'file')
+        disp('=== Downloading DLKcat docker image, this may take several minutes...')
```

Comment:
```suggestion
        disp('=== Downloading DLKcat docker image, this may take more than 20 minutes, depending on your internet connection speed...')
```